tests/Unit/App/Webhook/PdkWebhookManagerTest.php

```diff
@@ -21,18 +22,89 @@
 
 usesShared(new UsesMockPdkInstance(), new UsesMockEachCron());
 
-it('dispatches incoming webhook', function (string $hook) {
+it('dispatches and executes webhooks with myparcel header', function (string $hook) {
     /** @var PdkWebhooksRepositoryInterface $repository */
     $repository = Pdk::get(PdkWebhooksRepositoryInterface::class);
     /** @var PdkWebhookManagerInterface $webhookManager */
     $webhookManager = Pdk::get(PdkWebhookManagerInterface::class);
     /** @var \MyParcelNL\Pdk\Tests\Bootstrap\MockCronService $cronService */
     $cronService = Pdk::get(CronServiceInterface::class);
+    /** @var \MyParcelNL\Pdk\Tests\Bootstrap\MockLogger $logger */
+    $logger = Pdk::get(LoggerInterface::class);
 
+    $repository->storeHashedUrl('https://example.com/hashed-url');
+
+    $repository->store(new WebhookSubscriptionCollection([['hook' => $hook, 'url' => $repository->getHashedUrl()]]));
+
+    $time = time();
+
+    $request = Request::create(
+        $repository->getHashedUrl(),
+        Request::METHOD_POST,
+        [],
+        [],
+        [],
+        ['HTTP_X_MYPARCEL_HOOK' => $hook],
+        json_encode([
+            'data' => [
+                'hooks' => [
+                    ['event' => $hook],
+                ],
+            ],
+        ])
+    );
+
+    $response = $webhookManager->call($request);
+
+    $scheduled = $cronService->getScheduledTasks();
+    $timestamp = $scheduled->first()['timestamp'];
+
+    expect($response->getStatusCode())
```

Comment:
Je test helemaal niet of dit werkt en de webhook succesvol geparsed wordt, je bent nu alleen maar aan het kijken of er een webhook geregistreerd is. Er wordt nu niet getest of dat header/event stuk werkt. Vandaar ook dat het niet werkt in je code maar je tests wel slagen.